packages/orchestration/src/exos/cosmos-orchestration-account.js

```diff
@@ -579,16 +689,15 @@ export const prepareCosmosOrchestrationAccountKit = (
  */
 export const prepareCosmosOrchestrationAccount = (
   zone,
-  makeRecorderKit,
-  vowTools,
-  zcf,
+  { chainHub, makeRecorderKit, timerService, vowTools, zcf },
```

Comment:
good refactor. I suppose this should be the habit from the outset. powers tend to expand